### DISCUSSION TITLE
How to import JSON field in CSV

### DISCUSSION THREAD

**Discussion by thomasmcinnis** [AntlerVC/firetable#1420] [Support / Q&A] [answered]
Hey folks! 
In my firestore I have an array field. I note that you can't bulk import to array with CSV so I expect JSON is the appropriate replacement type.
I manually added JSON to a record with "0" "1" and so on as keys and happily it works in Firestore with the array data appearing correctly.

But now trying to import from CSV I'm stumped as to how JSON content needs to be formatted in a CSV to be imported! I have tried many different versions with wrapping in "" and [] and so on but nothing that I think should work does.

Here is an example of what the correctly filled value should be:
<img width="304" alt="image" src="https://github.com/rowyio/rowy/assets/88759292/d5d7ffc7-60d7-45de-b254-8784e65a5e28">

**Comment by Chadha93** [comment, marked as answer]
Hi @thomasmcinnis. While importing CSV or any table in Rowy it auto-detects the field type but you can override the the field type to JSON when importing your CSV. 

You can refer this doc - https://docs.rowy.io/import-export-data/import-csv#supported-field-types-for-import-via-csv.
Video - https://www.youtube.com/watch?v=sZrepVfLnwg&t=51s.

I invite you to join our [Discord](https://www.rowy.io/discord) and post this on the community support channel.
